index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <img src="images.svg\image_geometry_1 (1).svg" alt="" class="top-left">
+        <nav><img src="images.svg\ic_menu (1).svg" alt="" class="hamburger"></nav>
+    </header>
+    <main>
+        <section class="section-1">
+            <h1>Iheagwara<br>Augustine</h1>
+        <p>
+            Hello! I am a software developer! I can help you build a product, feature or website. Take a look of my works.
+If you like what you see and have a project you need coded, don’t hesitate and contact me.
+        </p>
+        <button class="Start-collab">Start collaboration</button>
+        </section>
+        <img src="images.svg\image_geometry_2 (2).svg" alt="" class="image-1">
```

Comment:
- [ ] Kindly make this a `background-image` in your style sheet rather than an image in the HTML markup. If an image is not part of the content (for aesthetic reasons) use a CSS `background-image.` [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info. 💻 👍

Reply:
I would advice you again, please don't use background images in the HTML file but always only in the CSS file, as mentioned in a previous comment.